local-cli/generator-android/templates/src/app/build.gradle

```diff
@@ -18,6 +19,12 @@ apply plugin: "com.android.application"
  *   // whether to bundle JS and assets in debug mode
  *   bundleInDebug: false,
  *
+ *   // whether to bundle JS and assets in a build variant (if configured).
+ *   // See https://sites.google.com/a/android.com/tools/tech-docs/new-build-system/user-guide#TOC-Build-Variants
+ *   // The configuration property is in the format 'bundleIn${productFlavor}${buildType}'
+ *   // bundleInStageAlpha: true,
+ *   // bundleInBeta: true,
```

Comment:
nit: Can you make these examples something like "free", "paid"? I got confused by what "stage alpha" was when I first saw the PR.